api/v1beta1/ansibletest_types.go

```diff
@@ -47,7 +47,7 @@ type AnsibleTestSpec struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	// +kubebuilder:validation:Optional
 	// Extra configmaps for mounting in the pod.
-	ExtraMounts []extraConfigmapsMounts `json:"extraMounts"`
+	ExtraMounts []extraConfigmapsMounts `json:"extraMounts,omitempty"`
```

Comment:
Also, I'm thinking since we need to update the PR one way or the other. Can you move the `extraConfigmapsMounts` struct to a separate file with a name `common.go`? It would be a nice start for parameters that are consumed in more CRs.

Reply:
And one more thing. Can we rename the `extraConfigmapsMounts.Name` parameter to `extraConfigmapsMounts.name` (lowercase `n` to keep the names consistent)?

`json:"Name"` -> `json:"name"`